EpikCord/client/http.py

```diff
@@ -25,7 +25,7 @@
 logger = getLogger("EpikCord.http")
 
 class File:
-    def __init__(self, *, filename: str, contents: IOBase, mime_type: Optional[str] = None):
+    def __init__(self, *, filename: str, contents: IOBase, mime_type: Optional[str] = None, spoiler: bool = False, ephemeral: bool = False, description: Optional[str] = None):
```

Comment:
This class accumulates fields

Reply:
Meaning what?

Reply:
It's should be rewritten

Reply:
But those attributes can be set, so I don't really see what you mean. Even if I rewrite it, it'd still be the same thing.